convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -380,12 +388,13 @@ def __call__(self):
     @unit_tests.mock(pkghandler, "install_gpg_keys",
                      DumbCallableObject())
     @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
+    @unit_tests.mock(pkghandler, "get_installed_pkgs_by_fingerprint",
+                     GetInstalledPkgsWithFingerprintMocked(data=['kernel']))
     def test_preserve_only_rhel_kernel(self):
         pkghandler.preserve_only_rhel_kernel()
 
         self.assertEqual(utils.run_subprocess.cmd, "yum update -y kernel")
```

Comment:
Create another test for `preserve_only_rhel_kernel()` to verify that branch for `loggerinst.critical` works as well

Reply:
Could probably rewrite that to be more clear. There isn't any tests right now covering what happens inside `preserve_only_rhel_kernel()` if `is_rhel_kernel_installed()` returns False

Can be checked with a new test mocking out `is_rhel_kernel_installed` to return False and assert `self.assertRaises(SystemExit, preserve_only_rhel_kernel)`